test/lit/exec/fuzzing-api.wast

```diff
@@ -118,6 +120,139 @@
    )
   )
  )
+
+ ;; CHECK:      [fuzz-exec] calling ref.calling
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 42]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 3.14159]
+ ;; CHECK-NEXT: [exception thrown: __private ()]
+ (func $ref.calling (export "ref.calling")
+  ;; This will emit some logging.
+  (call $call.ref
+   (ref.func $logging)
+  )
+  ;; This will trap.
+  (call $call.ref
+   (ref.null func)
+  )
+ )
+
+ ;; CHECK:      [fuzz-exec] calling ref.calling.catching
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 42]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 3.14159]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 0]
+ ;; CHECK-NEXT: [LoggingExternalInterface logging 1]
+ (func $ref.calling.catching (export "ref.calling.catching")
+  ;; This will emit some logging, then log 0 as we do not error.
+  (call $log-i32
+   (call $call.ref.catch
+    (ref.func $logging)
+   )
+  )
+  ;; The trap here is caught, and we'll log 1.
+  (call $log-i32
+   (call $call.ref.catch
+    (ref.null func)
+   )
+  )
+ )
+
+ (func $legal (param $x i32) (result i32)
+  ;; Helper for the function below. All types here are legal for JS.
+  (call $log-i32
+   (i32.const 12)
+  )
```

Comment:
If we log the param instead of an arbitrary constant, then the test will show that it is indeed 0, as expected.

Reply:
I added that logging (I prefer to keep unique loggings here to avoid 0/1 being confused with threw/didn't throw.)